3.11/release-notes-api-changes311.md

```diff
@@ -115,6 +115,11 @@ Enterprise Edition:
   Like the existing `geojson` Analyzer, but with an additional `format` property
   that can be set to `"latLngDouble"` (default), `"latLngInt"`, or `"s2Point"`.
 
+#### Query API
```

Comment:
Would be nice to add cross-reference links to the endpoints:
- [http/aql-query.html#returns-the-list-of-slow-aql-queries]
- [http/aql-query.html#returns-the-currently-running-aql-queries]

Reply:
Links will be added in DOC-407 to avoid conflicts.